Source/EasyNetQ/DefaultRpc.cs

```diff
@@ -131,22 +133,35 @@ private void OnConnectionRecovered(in ConnectionRecoveredEvent @event)
         if (@event.Type != PersistentConnectionType.Consumer)
             return;
 
-        var responseActionsValues = responseActions.Values;
-        var responseSubscriptionsValues = responseSubscriptions.Values;
+        List<KeyValuePair<RpcKey, ResponseSubscription>> subEntries = responseSubscriptions.ToList();
```

Comment:
@Pliner What do you think about reenterability of `OnConnectionRecovered` method? Can it happen in practice?